config/environments/production.rb

```diff
@@ -47,8 +47,10 @@
   # information to avoid inadvertent exposure of personally identifiable information (PII).
   config.log_level = :info
 
-  # Prepend all log lines with the following tags.
-  config.log_tags = [:request_id]
+  # https://technical-guidance.education.gov.uk/infrastructure/monitoring/logit/#ruby-on-rails
```

Comment:
Could you test with and without dfe analytics? In other services we saw it was flooding the logs and we filtered them out using:
https://github.com/DFE-Digital/teaching-vacancies/blob/main/config/environments/production.rb#L77